src/codeu/chat/server/Controller.java

```diff
@@ -164,6 +171,18 @@ private Uuid createId() {
     return candidate;
   }
 
+  @Override
+  public void renameConversation(Uuid id, String newTitle) {
+    final Conversation foundConv = model.conversationById().first(id);
+
+    if(foundConv != null && isIdInUse(id)) {
+
+      LOG.info("Conversation renamed: " + foundConv.title + " -- " + newTitle);
+      foundConv.changeTitle(newTitle);
```

Comment:
Rather than changing the title in the conversation, how about creating a new conversation with the new title and updating it in the model?